templates/scenarios/csharp/non-sso-tab/Properties/launchSettings.json.tpl

```diff
@@ -11,15 +11,16 @@
       },
       "hotReloadProfile": "aspnetcore"
     },
```

Comment:
Does the `,` break the json format in VS?

Reply:
It will show warning information, thanks for the idea.